consensus/ibft/ibft.go

```diff
@@ -626,3 +626,15 @@ func verifyProposerSeal(
 
 	return nil
 }
+
+// ValidateExtraDataFormat Verifies that extra data can be unmarshaled
+func (i *backendIBFT) ValidateExtraDataFormat(header *types.Header) error {
+	blockSigner, _, _, err := getModulesFromForkManager(
```

Comment:
shall we check this error here?

Reply:
Yes I feel like we actually should. Thank you